packages/main/src/ListItemBase.js

```diff
@@ -75,6 +87,12 @@ class ListItemBase extends UI5Element {
 		return styles;
 	}
 
+	onBeforeRendering() {
+		if (this.disabled) {
```

Comment:
Yes, this is exactly the case when a focused item is disabled.

Did you mean replacing the onBeforeRendering logic in ListItemBase.js with more precise CSS selectors? 
=>
I started this way, but then noticed that the presence of the 'focused' attribute [on a disabled list-item] does not actually cause any visual issue (it has no effect anymore because a disabled list-item does not have the 'ui5-li--focusable' class) => no visual issue there => I cancelled modifying the selectors for the focused items

Unfortunately, there was still a different issue (unsolved in my previous code):
Issue:
If we disable a focused list-item, the browser does not automatically remove the focus outline from the disabled item, even after the disabled item receives tabindex=-1 it DOM
=> I was able to remove the outline only after calling blur()
=> I updated the code to call blur() in the above case onAfterRendering 

I pondered really a lot which way to go.. No problem to change the solution if you have a better idea in mind! I'm around for a call for quicker clarifications if needed! Thnx!

Reply:
In my latest commit I made a different solution for this issue:
added some code to onAfterRendering of the Select:
In this way,  if a focused list-item was disabled while the picker was still open , then onAfterRendering we ensure the focus is moved to the newly selected item
=> the browser no longer keeps the focus on the disabled item

Let me know if you have a better idea/see ways to improve on that.

Reply:
Yes, I meant replacing the onBeforeRendering logic in ListItemBase.js with more precise CSS selectors in my comment.
I will see the latest changes as soon as possible and let you know.